.editorconfig

```diff
@@ -0,0 +1,13 @@
+# editorconfig.org
+root = true
+
+[*]
+indent_style = space
+indent_size = 2
+end_of_line = lf
+charset = utf-8
+trim_trailing_whitespace = true
+insert_final_newline = true
```

Comment:
Are we 100% sure all of these are compatible with our eslint/prettier config? 👀 
Wouldn't just `end_of_line = lf` be enough?

Reply:
I can remove `indent_size`, other stuff I will keep

Reply:
It's fine then